openmetadata-service/src/main/java/org/openmetadata/service/resources/teams/UserResource.java

```diff
@@ -483,7 +483,7 @@ public User getVersion(
       })
   public Response createUser(
       @Context UriInfo uriInfo, @Context SecurityContext securityContext, @Valid CreateUser create) {
-    User user = getUser(securityContext, create);
+    User user = getUser(securityContext.getUserPrincipal().getName(), create);
```

Comment:
why are we changing this?

Reply:
we were passing the while SecurityPrincipal just updatedBy field is required